### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by ming-00**
- Added completed challenge page
 - Added placeholder badge page
 - Fetch and display ongoing & completed challenges

Here is completed challenge page ss so you don't have to go through the trouble of seeding 😅 
Screenshot 2021-10-17 at 9 35 45 PM

### CODE REVIEW

File: backend/app/controllers/user_challenges_controller.rb [hunk 1]
```diff
@@ -10,4 +10,8 @@ def ongoing_user_challenge
 
     @user_tasks = current_user.user_tasks.includes(:task).where(user_challenge_id: @user_challenge.id)
   end
+
+  def all_ongoing_challenges
+    @user_challenge = current_user.user_challenges.find_by(completed_at: nil, forfeited_at: nil)
```
Comment (sebastiantoh) [inline backend/app/controllers/user_challenges_controller.rb:15]: hmm, is this supposed to retrieve > 1 user challenge? If so, I think you should probably replace `find_by` with `where` and rename the variable to be plural

Comment (ming-00) [reply backend/app/controllers/user_challenges_controller.rb:15]: Oh yes thanks! I changed it already 😅 Wish I saw your comment sooner

File: frontend/src/utils/formatting.ts [hunk 2]
```diff
@@ -53,3 +54,9 @@ export const getFormattedStringFromDays = (numberOfDays: number): string => {
   const daysDisplay = days > 0 ? days + (days == 1 ? ' day' : ' days') : '';
   return yearsDisplay + monthsDisplay + daysDisplay;
 };
+
+export const convertTimestampToHumanReadableFormat = (
+  timestamp: number
+): string => {
+  return moment(timestamp).utc().format('MMMM Do YYYY, h:mm a');
```
[the remark this answers is not in the document]
Comment (ming-00) [reply frontend/src/utils/formatting.ts:61]: Ah okay!

**Review by sebastiantoh — APPROVED** [review]
lgtm!

**Pull request merged**